packages/strapi-plugin-graphql/hooks/graphql/index.js

```diff
@@ -119,6 +119,7 @@ module.exports = strapi => {
         bodyParserConfig: true,
         introspection: _.get(strapi.plugins.graphql, 'config.introspection', true),
         engine: _.get(strapi.plugins.graphql, 'config.engine', false),
+        cacheControl: _.get(strapi.plugins.graphql, 'config.cacheControl', {}),
```

Comment:
Hey, Instead of adding every single key one at a time I suggest with just omit the fields we don't want to pass down and pass the rest of the properties so it allows passing any possible properties :)

Reply:
And what do you want to omit?

Reply:
We need to review that but I think theses ones:

shadowCRUD
playgroundAlways
depthLimit
amountLimit

Reply:
@alexandrebodin something like this? #7665